modules/container_deployment/main.tf

```diff
@@ -71,14 +67,38 @@ resource "helm_release" "jaeger-operator" {
 
   repository = "https://jaegertracing.github.io/helm-charts"
   chart      = "jaeger-operator"
+  values = [
+    file("${path.module}/jaeger_values.yaml")
+  ]
+}
 
-  set {
-    name  = "jaeger.create"
-    value = "true"
+# Basically copied from Hono Helm charts
+resource "kubernetes_config_map" "grafana_hono_dashboards" {
+  metadata {
+    name    = "grafana-hono-dashboards"
+    labels  = {
+      "grafana_dashboard": "1" # Add labels so that Grafana finds these
+    }
   }
 
-  set {
-    name  = "metadata.name"
-    value = "simple"
+  data = {
+    "jvm-details.json"      = file("${path.module}/Grafana_Dashboards/JVM_details_grafana_ds.json")
+    "message-details.json"  = file("${path.module}/Grafana_Dashboards/Message_details_grafana_ds.json")
+    "overview.json"         = file("${path.module}/Grafana_Dashboards/Overview_grafana_ds.json")
   }
 }
+
+# https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack
+# Values that can be set: "helm show values prometheus-community/kube-prometheus-stack" (repo has to be added first)
```

Comment:
I recommend using \`$ helm show values prometheus-community/kube-prometheus-stack` when referring to terminal / shell commands.

Reply:
Good point. Fixed.